styles/_main.scss

```diff
@@ -0,0 +1,72 @@
+main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-top: 1rem;
+}
+
+// .row {
+//   display: flex;
+//   // justify-content: flex-end;
+//   // overflow: auto;
+//   // width: 75%;
+//   margin-top: $gutter;
+// }
+
+.main__article {
+   display: flex;
+   justify-content: flex-end;
+   margin: auto;
+
+  //  padding-left: 5vw
+}
+
+.pic-and-author {
+  flex-basis: calc(60% - 2rem);
```

Comment:
If you don't have margin, this `-2rem` is going to mess with things centering up on the screen